sdk/template/azure-template-stress/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG REGISTRY="azsdkengsys.azurecr.io"
+FROM ${REGISTRY}/java/jdk-mariner-mvn:jdk11-latest as builder
+
+RUN yum -y update
```

Comment:
It could be good to emphasize not removing this line.

```suggestion
# Do not remove this line. Update ensures container images do not get flagged for out of date and vulnerable distro packages.
RUN yum -y update
```